api/v1alpha1/inferencepool_types.go

```diff
@@ -59,8 +59,88 @@ type InferencePoolSpec struct {
 	// +kubebuilder:validation:Maximum=65535
 	// +kubebuilder:validation:Required
 	TargetPortNumber int32 `json:"targetPortNumber"`
+
+	// EndpointPickerConfig specifies the configuration needed by the proxy to discover and connect to the endpoint
+	// picker service that picks endpoints for the requests routed to this pool.
+	EndpointPickerConfig `json:"endpointPickerConfig"`
```

Comment:
Since this is an embedded field, should `inline` be used in the json tag? From the [json serialization pkg](https://pkg.go.dev/github.com/go-json-experiment/json):

> nline: The "inline" option specifies that the JSON representable content of this field type is to be promoted as if they were specified in the parent struct. It is the JSON equivalent of Go struct embedding. A Go embedded field is implicitly inlined unless an explicit JSON name is specified...

This would be consistent with `TypeMeta`.

Reply:
done.